libs/common/src/enums/event-system-user.ts

```diff
@@ -0,0 +1,3 @@
+export enum EventSystemUser {
```

Comment:
I think it's convenient to use the enum key in the UI, but it's unusual. A developer would not generally expect this (usually it's just for our convenience when reading the code), so I think we should flag it explicitly.

```suggestion
// Note: the enum key is used to describe the EventSystemUser in the UI. Be careful about changing it.
export enum EventSystemUser {
```